src/jobs/schemas/job.schema.ts

```diff
@@ -107,7 +107,7 @@ export class JobClass extends OwnableClass {
     type: Object,
     required: false,
   })
-  configuration: Record<string, unknown>;
+  configuration: Record<string, unknown>;  // TBD
```

Comment:
In the schema we store `jobParams`, so that is later accessible when performing an update.
1. Should we also store the configuration, and if yes why?
2. Should job actions be also somehow stored in the db? Yes/no and why?

Reply:
If space and effort allows, I would store both of them.
I would do that for validation purposes. The job definition can change, so it would be nice to know what was the definition when the job was created.

Reply:
1. Configuration should not be stored in the db. We can keep track of the config version (i.e. hash) and if config gets  updated, we stop all running jobs and restart.